### PR TITLE
[FIX] web: reintroduce ka*N*ban record action button hover style

### PR DESCRIPTION
**Pull request by SimonGenin**

### CODE REVIEW

**Comment by aab-odoo** [comment]
robodoo r+

**Comment by pparidans** [comment]
@robodoo r-

Sorry, it mess the color in community

File: addons/web/static/src/views/kanban/kanban_controller.scss [hunk 1]
```diff
@@ -661,3 +661,9 @@
         max-height: 100%;
     }
 }
+
+// Kanban record tile action button
+.oe_kanban_action.oe_kanban_action_a span:hover {
+    background-color: #017e84;
```
Comment (pparidans) [inline addons/web/static/src/views/kanban/kanban_controller.scss:667]: Use the SCSS variables to adapt to the right color (community vs enterprise)

**Comment by pparidans** [comment]
@robodoo r+

**Pull request closed without merging**